lib/mayaUsd/render/vp2RenderDelegate/draw_item.h

```diff
@@ -59,6 +59,12 @@ class HdVP2DrawItem final : public HdDrawItem
         //! World matrix of the render item.
         MMatrix _worldMatrix;
 
+        //! Instance transforms for the render item
+        std::shared_ptr<MMatrixArray> _instanceTransforms;
+
+        //! Instance colors for the render item
+        std::shared_ptr<MFloatArray> _instanceColors;
+
```

Comment:
Store these as shared_ptr so I can easily transfer ownership of the data from the commit state to the draw item data.